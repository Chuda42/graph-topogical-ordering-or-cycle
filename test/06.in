crear_personas 8
leer_lista_muertes 3 1 6 1 6 2 8 3 8 4 .
leer_lista_convivencias 1 2 1 5 3 4 3 6 5 8 6 8 7 8 .
existe_cronologia
Fin