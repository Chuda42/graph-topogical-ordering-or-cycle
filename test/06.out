1>2>3>4>Hay una cronologÃ­a consistente.
5>Fin.
